scripts/start.sh

```diff
@@ -75,7 +75,4 @@ docker exec $DB_ANNEX_CONTAINER montagu-wait.sh
 docker run --rm --network=$NETWORK $MIGRATE_IMAGE -configFile=conf/flyway-annex.conf migrate
```

Comment:
Should we be bringing up annex at all in this script now? Similar issues in scripts/stop.sh and scripts/test-migrate.sh

Reply:
Do we know where this script is in use? It is used by the api, but anywhere else?

Reply:
that's all I can fine